client/human/HumanClientApp.cpp

```diff
@@ -1127,3 +1128,44 @@ HumanClientApp* HumanClientApp::GetApp()
 
 void HumanClientApp::Initialize()
 {}
+
+void HumanClientApp::OpenURL(const std::string& url) {
```

Comment:
The point of the tests is to make sure it's not a link to a local file, so that it can't be used as a way to execute arbitrary code locally if used from within a pedia link (not yet implemented) If you want to add a much more complicated test, feel free to suggest something, but I think these tests serve my purpose.